exchange/filecoinretrieval/filecoinretrieval.go

```diff
@@ -48,31 +49,44 @@ type VoucherCreateResult struct {
 	Shortfall big.Int
 }
 
-type Node interface {
+type PaymentAPI interface {
 	GetPaychWithMinFunds(ctx context.Context, dest address.Address) (address.Address, error)
```

Comment:
Typo `Paych`?

Reply:
nope, this would be an external API to something external that handles wallets and makes paymentchannels.